extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/CharArrayTransfer.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.parquet.table.transfer;
+
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ColumnSource;
+import org.jetbrains.annotations.NotNull;
+
+import java.nio.IntBuffer;
+
+final class CharArrayTransfer extends PrimitiveArrayAndVectorTransfer<char[], char[], IntBuffer> {
+    CharArrayTransfer(@NotNull final ColumnSource<?> columnSource, @NotNull final RowSequence tableRowSet,
+                      final int targetPageSize) {
+        // We encode characters as integers
```

Comment:
I've noticed that this comment carried through (incorrectly) to `ByteArrayTransfer`. I think it might replicate cleanly this way, but you should double check.
```suggestion
        // We encode primitive chars as primitive ints
```